src/plugins/console/server/routes/api/console/spec_definitions/index.ts

```diff
@@ -32,5 +32,4 @@ export const registerSpecDefinitionsRoute = ({ router, services }: RouteDependen
   };
 
   router.get({ path: '/api/console/api_server', validate: false }, handler);
-  router.post({ path: '/api/console/api_server', validate: false }, handler);
```

Comment:
I don't think this route is used with `POST`, according to this [file](https://github.com/elastic/kibana/blob/11e57be842e5ea22076e6fe7e654f31bb18f6e90/src/plugins/console/public/lib/kb/kb.js#L122), it's called via an `ajax` request that defaults to `GET`.